diff --git a/vcl/CMakeLists.txt b/vcl/CMakeLists.txt
index dd83858d..70db5372 100644
--- a/vcl/CMakeLists.txt
+++ b/vcl/CMakeLists.txt
@@ -115,13 +115,12 @@ vxl_configure_file(${CMAKE_CURRENT_LIST_DIR}/vcl_where_root_dir.h.in ${PROJECT_B
 
 include_directories(${PROJECT_BINARY_DIR})
 set( vcl_sources
-  vcl_legacy_aliases.h ${VCL_COMPILER_DETECTION_HEADER}
-  vcl_deprecated.cxx  vcl_deprecated.h
-
+  vcl_legacy_aliases.h
+  vcl_deprecated.cxx
+  vcl_deprecated.h
   vcl_cmath.h
   vcl_cassert.h
   vcl_cstdlib.h
-
   vcl_algorithm.h
   vcl_bitset.h
   vcl_cctype.h
@@ -156,6 +155,7 @@ set( vcl_sources
   vcl_locale.h
   vcl_map.h
   vcl_memory.h
+  vcl_msvc_warnings.h
   vcl_numeric.h
   vcl_ostream.h
   vcl_queue.h
